.github/workflows/consistency-functional.yml

```diff
@@ -263,6 +263,10 @@ jobs:
       - name: Clone devstack
         run: |
           git clone --single-branch --branch master https://opendev.org/openstack/devstack
+          # Clone from current Devstack WIP review
+          cd devstack
+          git fetch https://review.opendev.org/openstack/devstack refs/changes/07/823807/1 && git checkout -b change-823807 FETCH_HEAD
```

Comment:
This is testing the upstream review that fixes our CI let's wait for the Devstack folks to review the patch before merging this, so we can remove the workaround.